app/src/main/java/org/mozilla/reference/browser/browser/BrowserFragment.kt

```diff
@@ -106,11 +109,14 @@ class BrowserFragment : Fragment(), BackHandler {
             sessionId, ::fullScreenChanged
         )
 
+        pipFeature = PictureInPictureFeature(requireComponents.core.sessionManager, activity, this)
```

Comment:
nit: You may want to pass in `requireActivity` here to not have to deal with nullability.